sdk/go/auto/local_workspace_test.go

```diff
@@ -1727,20 +1727,34 @@ func TestEnvFunctions(t *testing.T) {
 	require.NoError(t, s.AddEnvironments(ctx, "automation-api-test-env", "automation-api-test-env-2"),
 		"adding environments failed, err: %v", err)
 
+	envs, err := s.ListEnvironments(ctx)
+	require.NoError(t, err, "listing environments failed, err: %v", err)
+	assert.Equal(t, 2, len(envs))
+	assert.Equal(t, "automation-api-test-env", envs[0])
+	assert.Equal(t, "automation-api-test-env-2", envs[1])
```

Comment:
Nit:

```suggestion
	assert.Equal(t, []string{"automation-api-test-env", "automation-api-test-env-2"}, envs)
```

And similar below.